src/EnergyPlus/IOFiles.cc

```diff
@@ -319,6 +319,14 @@ std::vector<std::string> InputOutputFile::getLines()
     return std::vector<std::string>();
 }
 
+bool IOFiles::OutputControl::writeTabular(EnergyPlusData &state)
+{
+    bool const htmlTabular = state.files.outputControl.tabular;
+    bool const jsonTabular = state.files.outputControl.json && state.dataResultsFramework->resultsFramework->timeSeriesAndTabularEnabled();
+    bool const sqliteTabular = state.files.outputControl.sqlite; // && @JasonGlazer thinks something else maybe?
+    return (htmlTabular || jsonTabular || sqliteTabular);
+}
+
```

Comment:
Can someone please explain what the benefit of the above is compared to a one line:

    return (state.files.outputControl.tabular || (state.files.outputControl.json && state.dataResultsFramework->resultsFramework->timeSeriesAndTabularEnabled()) || state.files.outputControl.sqlite);

Reply:
To the compiler? Nothing. To the person trying to read that one long line and track the operators and parentheses depth? I think it's tough to read the long line.